CHANGELOG.md

```diff
@@ -20,4 +18,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please see PRs for specifications before changelog
 
 [unreleased]: https://github.com/teamsempo/SempoBlockchain/compare/v1.7.70...HEAD
```

Comment:
Delete

Reply:
@tristanhcole Do'h! I accidentally made this change in a 2nd branch while on the call! lol!